WTWSMS/decisions/VIET_religion_decisions.txt

```diff
@@ -987,12 +997,22 @@ decisions = {
 		only_playable = yes
 		potential = {
 			has_landed_title = k_israel
-			religion_group = jewish_group
+			OR = {
+				religion_group = jewish_group
+				religion = ebionites
+			}
 			NOT = { trait = house_of_david }
 			NOT = { has_dynasty_flag = claimed_descent_from_david }
 		}
 		allow = {
-            piety = 200
+		            piety = 200
+			hidden_tooltip = { # Character must be ethnically Hebrew (AKA graphical culture AKA physical appearence, not culture) as David, or
+#					# passable by ancient norms to be Hewbrew (since no DNA tests exist, Sephardis could LARP as Hebrews if they were lucky)
+				text = MUSTHEBREW
+
+				race = hebrew
+				race = sephardi
+			}
```

Comment:
Wrong.

I am on the editing board of the Avatar Mod, and one feature we use is that characters with ancestors of a certain race (not culture, but actual GFX) can become Elemental Benders themselves. It represents ancestry in a way that no other means can.

You are right, though, that it's too strict. I'll make it so that it also counts parents and grandparents.